src/components/ListBooks.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import Book from './Book';
+
+function Books({ books }) {
+  return (
+    <>
+
+      {
+        books.map((book) => (
```

Comment:
[OPTIONAL]
- I would suggest adding a key prop for each child component in your mapping because currently, you get the error seen below on your console.
![image](https://github.com/mariegrace31/bookstore/assets/45460192/19076d54-2ff2-4afe-a7ee-ae4d45909966)

Reply:
Hi @Lucas-Masaba, Thanks for your review.